R/slide.R

```diff
@@ -88,6 +88,13 @@
 #' @importFrom lubridate days weeks
 #' @importFrom rlang .data .env !! enquo enquos sym
 #' @export
+#' @examples 
+#' library(dplyr)
+#' 
+#' # slide a 7-day trailing average formula on cases
```

Comment:
This is fine, but maybe include some other examples using the `align` and `as_list_col` arguments?

Reply:
sounds good. I'll add those as well.